scripts/check_for_update.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
Your call but maybe this is a good time to migrate any large shell script files to js.